packages/loader/container-loader/src/container.ts

```diff
@@ -1992,6 +1993,7 @@ export class Container
 		batch?: boolean,
 		metadata?: any,
 		compression?: string,
+		referenceSequenceNumber?: number,
```

Comment:
I'd not worry about this path - that's legacy path used by old runtimes, that will not provide this info any way, so there is not much point to add it here.

Reply:
Not sure if it's the legacy path. This is used by `private submitBatch(batch: IBatchMessage[]): number` above.

Reply:
Correct - it's feature detection. Runtime will try to use submitBatchFn() if it exists, otherwise switch to submitFn()
The version of runtime that has support for refSeqNumber will have submitBatchFn(), so runtime will chose it.
The exception might be for some stand-alone ops (like summarize Op), but I'm not sure it matters.
Double check these claims, please :)

Reply:
That's correct. And if we want to give the referenceSequenceNumber to the loader, we will need it in the signature of this method, as it is used by submitBatchFn and the latter is the entry-point from the runtime.